mindsdb/integrations/handlers/replicate_handler/Manual_QA.md

```diff
@@ -0,0 +1,69 @@
+# Welcome to the MindsDB Manual QA Testing for Replicate Handler
+
+> **Please submit your PR in the following format after the underline below `Results` section. Don't forget to add an underline after adding your changes i.e., at the end of your `Results` section.**
+
+## Testing Replicate Handler 
+
+**1. CREATE MODEL**
+
+```sql
+CREATE MODEL stability
+PREDICT url
+USING
+    engine = 'replicate',
+    model_name= 'ai-forever/kandinsky-2',
+    version ='601eea49d49003e6ea75a11527209c4f510a93e2112c969d548fbb45b9c4f19f',
+    api_key = 'r8_BpOZN..............................';
+```
+
+**2. DESCRIBE PREDICTOR to see available parameters**
+
+```sql
+DESCRIBE PREDICTOR mindsdb.aiforever.features;
+```
+
+```sql
++---------------------+-------------------+--------------------------------------------------------+---------+
+| inputs              | default           | description                                            | type    |
++---------------------+-------------------+--------------------------------------------------------+---------+
+| width               | 512               | Choose width. Lower the setting if out of memory.      | -       |
+| height              | 512               | Choose height. Lower the setting if out of memory.     | -       |
+| prompt              | red cat, 4k photo | Input Prompt                                           | string  |
+| scheduler           | p_sampler         | Choose a scheduler                                     | -       |
+| batch_size          | 1                 | Choose batch size. Lower the setting if out of memory. | -       |
+| prior_steps         | 5                 | -                                                      | string  |
+| guidance_scale      | 4                 | Scale for classifier-free guidance                     | number  |
+| prior_cf_scale      | 4                 | -                                                      | integer |
+| num_inference_steps | 50                | Number of denoising steps                              | integer |
++---------------------+-------------------+--------------------------------------------------------+---------+
+```
+
+**3. Testing SELECT FROM PREDICTOR**
+
+```sql
+SELECT * FROM aiforever

```

Comment:
Same as above, seems like the example has created a model called `stability`, not `aiforever`, please double check.

Reply:
done